src/main/java/net/tiagofar78/prisonescape/bukkit/BukkitItems.java

```diff
@@ -0,0 +1,109 @@
+package net.tiagofar78.prisonescape.bukkit;
+
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+
+import net.tiagofar78.prisonescape.game.PrisonEscapeItem;
+
+public class BukkitItems {
+	
+	public static PrisonEscapeItem convertToPrisonEscapeItem(ItemStack item) {
+		if (item == null) {
+			return null;
+		}
+		
+		switch (item.getType()) {
+		case SPYGLASS:
+			return PrisonEscapeItem.SEARCH;
+		case IRON_BARS:
+			return PrisonEscapeItem.HANDCUFS;
+		case WOODEN_SHOVEL:
+			return PrisonEscapeItem.PLASTIC_SPOON;
+		case GOLDEN_SHOVEL:
+			return PrisonEscapeItem.METAL_SPOON;
+		case STONE_SHOVEL:
+			return PrisonEscapeItem.PLASTIC_SHOVEL;
+		case DIAMOND_SHOVEL:
+			return PrisonEscapeItem.METAL_SHOVEL;
+		case TORCH:
+			return PrisonEscapeItem.MATCHES;
+		case IRON_NUGGET:
+			return PrisonEscapeItem.BOLTS;
+		case SLIME_BALL:
+			return PrisonEscapeItem.DUCTTAPE;
+		case STRING:
+			return PrisonEscapeItem.WIRE;
+		case BOWL:
+			return PrisonEscapeItem.PLASTIC_PLATE;
+		case FLINT:
+			return PrisonEscapeItem.OIL;
+		case STICK:
+			return PrisonEscapeItem.STICK;
+		case POTION:
+			return PrisonEscapeItem.COFFEE;
+		default:
+			return null;
+		}
+	}
+	
+	public static ItemStack convertToItemStack(PrisonEscapeItem item) {
+		if (item == null) {
```

Comment:
indentation is off here. Not sure why, you might have the same problem I have

Reply:
It has 2 tabs, like everywhere in the code. Does it look different on vscode? I think that's just github making a tab look really big.

Reply:
Did check on vscode, just on github

Reply:
Then I think it will be fine, there are a lot of times that identation in github looks different from what it looks in editors.